src/main/webapp/app/exercises/programming/manage/update/programming-exercise-update.component.ts

```diff
@@ -625,11 +625,19 @@ export class ProgrammingExerciseUpdateComponent implements OnInit {
     }
 
     private onSaveError(error: HttpErrorResponse) {
-        const errorMessage = error.headers.get('X-artemisApp-alert')!;
+        let errorMessage;
+        let disableTranslation = true;
+        // Workaround for conflict error, since conflict errors do not have the 'X-artemisApp-alert' header
+        if (error.status === 409 && error.error && error.error['X-artemisApp-error'] === 'error.sourceExerciseInconsistent') {
+            errorMessage = 'artemisApp.consistencyCheck.error.programmingExerciseImportFailed';
+            disableTranslation = false;
+        } else {
+            errorMessage = error.headers.get('X-artemisApp-alert')!;
+        }
```

Comment:
```suggestion
        let errorMessage = error.headers.get('X-artemisApp-alert')!;
        let disableTranslation = true;
        // Workaround for conflict error, since conflict errors do not have the 'X-artemisApp-alert' header
        if (error.status === 409 && error.error && error.error['X-artemisApp-error'] === 'error.sourceExerciseInconsistent') {
            errorMessage = 'artemisApp.consistencyCheck.error.programmingExerciseImportFailed';
            disableTranslation = false;
        }
```
Or the other way around. But currently, you handle assigning values in two different ways here.